### PR TITLE
Molecules/table

### PR DESCRIPTION
**Pull request by majiiid-drykiss**

### CODE REVIEW

File: components/molecules/pageLoading/components/pageLoading.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
+
+PageLoading.defaultProps = {
+  backgroundColor: '#000',
+  opacity: 0.5,
+  position: 'fixed'
 }
 
 const StyledLoading = styled.div`
-  position: fixed;
+  position: ${({ position }) => position};
```
Comment (iwarner) [inline components/molecules/pageLoading/components/pageLoading.js]: @majiiid-drykiss Alpha these please

File: components/molecules/table/__tests__/stories.js [hunk 2]
```diff
@@ -67,3 +67,5 @@ export const formatter = () => <Table columns={columnsFormatter} rows={data} />
 export const actions = () => <Table columns={columnsActions} rows={data} />
 
 export const withPagination = () => <Table columns={columnsActions} rows={data} pagination />
+
+export const loading = () => <Table columns={columns} rows={data} loading />
```
Comment (iwarner) [inline components/molecules/table/__tests__/stories.js]: @majiiid-drykiss Give me a story with loading where the columns and rows are not defined yet

File: components/molecules/table/components/row.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  ${({ context, striped, theme }) =>
+    !context &&
+    striped &&
+    css`
+      :nth-child(odd) {
+        background-color: ${theme.COLOUR.light};
+      }
+    `}
+
+  ${props => props.context && BACKGROUND(props)}
```
Comment (iwarner) [inline components/molecules/table/components/row.js]: @majiiid-drykiss Cant this go into the block below also

File: components/molecules/table/components/table.js [hunk 4]
```diff
@@ -20,6 +27,8 @@ export const Table = ({
   className,
   columns,
   hover,
+  loading,
+  noData,
```
Comment (iwarner) [inline components/molecules/table/components/table.js]: @majiiid-drykiss We have to explicitly say noData? should it not be implied from the loading?

File: components/molecules/pageLoading/components/pageLoading.js (hunk 1 above)
Comment (majiiid-drykiss) [reply components/molecules/pageLoading/components/pageLoading.js]: Done

File: components/molecules/table/__tests__/stories.js (hunk 2 above)
Comment (majiiid-drykiss) [reply components/molecules/table/__tests__/stories.js]: Done

File: components/molecules/table/components/table.js (hunk 4 above)
Comment (majiiid-drykiss) [reply components/molecules/table/components/table.js]: Done

File: components/molecules/table/components/row.js (hunk 3 above)
Comment (majiiid-drykiss) [reply components/molecules/table/components/row.js]: Done

**Pull request merged**